cds_ils/migrator/relations/api.py

```diff
@@ -116,25 +116,30 @@ def migrate_document_siblings_relation():
                     document_class, legacy_pid_type, relation["related_recid"]
                 )
             except PIDDoesNotExistError as e:
-                pass
-
-            # try to find sibling in series
-            if related_sibling is None:
-                try:
-                    related_sibling = get_record_by_legacy_recid(
-                        series_class,
-                        legacy_pid_type,
-                        relation["related_recid"],
+                continue
+            rec_type = related_sibling["$schema"].split("/")[-1].split("-")[0]
+            # If it's series it must be a multipart, so we search for the
+            # first volume and link it.
+            if rec_type == "series":
+                doc_search = document_search.query(
+                    "query_string",
+                    query='relations.multipart_monograph.pid_value:"{}"'
+                                            .format(related_sibling["pid"])
+                )
+                doc_result = doc_search.execute()
+                if doc_result.hits.total.value > 0:
```

Comment:
is it ok to have for example 2 of them? More than 1?

Reply:
Yes, Anne said to match it with the first one created.